add-chain/e2e_test.go

```diff
@@ -93,15 +102,15 @@ func TestAddChain_Main(t *testing.T) {
 
 	t.Run("compress-genesis", func(t *testing.T) {
 		// Must run this test to produce the .json.gz output artifact for the
-		// subsequent CheckGenesisConfig test
+		// subsequent CheckGenesis test
```

Comment:
Would this be more accurate?
```suggestion
		// subsequent TestAddChain_CheckGenesis invocation. 
```